apps/documentation/src/components/MdxProvider/MdxTitle/MdxTitle.tsx

```diff
@@ -1,15 +1,52 @@
-import type { ElementType, JSX, ReactNode } from 'react'
-import { Title, type TitleProps } from '@mantine/core'
+import {
+  useCallback,
+  type ElementType,
+  type JSX,
+  type MouseEvent,
+  type ReactNode,
+} from 'react'
+import { Title, Anchor, type TitleProps } from '@mantine/core'
+import { IconLink } from '@tabler/icons-react'
+import { useHover } from '@mantine/hooks'
 
 export default function MdxTitle(props: TitleProps): JSX.Element {
```

Comment:
Also I think that `order` should be renamed to `level` to match HTML:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements
   
   > The `<h1>` to `<h6>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) elements represent six levels of section headings. `<h1>` is the highest section level and `<h6>` is the lowest.

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level#with_heading_role

Reply:
Order is a mantine `TitleProps` field. I wouldn't change it